frontend/src/views/eligibility/EligibilityHome.vue

```diff
@@ -5,16 +5,21 @@ import SubNavTab from '../../components/template/SubNavTab.vue'
 
 <template>
   <TheSubNav>
-    <SubNavTab routeName="CheckEligibility" title="Check Eligibility" />
+    <SubNavTab routeName="CheckEligibility" title="Check Eligibility" v-if="hasPermission('R15')" />
     <SubNavTab routeName="PhnInquiry" title="PHN Inquiry" />
     <SubNavTab routeName="PhnLookup" title="PHN Lookup"/>
-    <SubNavTab routeName="CoverageStatusCheck" title="MSP Coverage Status Check" />
+    <SubNavTab routeName="CoverageStatusCheck" title="MSP Coverage Status Check" v-if="hasPermission('E45')" />
   </TheSubNav>
   <router-view />
 </template>
 
 <script>
 export default {
   name: 'EligibilityHome',
+  methods: {
```

Comment:
is it possible/worth-trying to define this globally so it can just be called directly in the template rather than creating it in TheNavBar and in each View

Reply:
I'd like to update from the current version (4.0.2) of Vuex to Pinia (which is essentially Vuex 5) which simplifies store management. When I do so I will update to use something like mapGetters which will avoid boilerplate functions to wrap calls to store getters.